packages/astro/src/@types/astro.ts

```diff
@@ -1587,6 +1617,15 @@ export interface AstroUserConfig {
  */
 export type InjectedScriptStage = 'before-hydration' | 'head-inline' | 'page' | 'page-ssr';
 
+/**
+ * IDs for different priorities of injected routes and redirects:
+ * - "normal": Merge with discovered file-based project routes, behaving the same as if the route
+ *   was defined as a file in the project.
+ * - "legacy": Use the old ordering of routes. Inject routes will override any file-based project route,
+ *   and redirects will be overriden by any project route on conflict.
```

Comment:
typo: `overriden` -> `overridden`
```suggestion
 *   and redirects will be overridden by any project route on conflict.
```